src/api/Metadata.js

```diff
@@ -354,22 +352,118 @@ class Metadata extends File {
         return editors;
     }
 
+    /**
+     * Utility to concat instance and template into one entity.
+     *
+     * @param {Object} instance - metadata instance
+     * @param {Object} template - metadata template
+     * @return {Object} metadata template instance
+     */
+    createTemplateInstance(instance: MetadataInstanceV2, template: MetadataTemplate): MetadataTemplateInstance {
+        const metadataFields: MetadataInstanceTemplateField[] = [];
+
+        // templateKey is unique identifier for the template,
+        // but its value is set to 'properties' if instance was created using Custom Metadata option instead of template
+        const isCustomMetadataInstance = template.templateKey !== METADATA_TEMPLATE_PROPERTIES;
```

Comment:
based on your comment, the opposite would be true

Reply:
the rest of the logic looks fine, but the name of variable is misleading

Reply:
Updated the name